### PR TITLE
Perform power analysis for CP features

### PR DESCRIPTION
**Pull request by gwaybio**
I also add a visualization of the power analysis, and add a visualization of the distribution of the top ER feature (since we are zooming in on that cluster of ER features)

### CODE REVIEW

File: 5_analyze_data/notebooks/KS-test_analysis/visualize_ks_test.r [hunk 1]
```diff
@@ -4,7 +4,7 @@ suppressPackageStartupMessages(library(dplyr))
 output_fig_dir <- "figures"
 
 ks_test_fig <- file.path(output_fig_dir, "ks_test_cp_genotype.png")
-actin_distrib_fig <- file.path(output_fig_dir, "top_actin_feature_distrib.png")
+actin_distrib_fig <- file.path(output_fig_dir, "top_er_feature_distrib.png")
```
Comment (jenna-tomkinson) [inline 5_analyze_data/notebooks/KS-test_analysis/visualize_ks_test.r]: Wouldn't you want to change the name of the variable to `er_distrib_fig`? This is nit picky so it doesn't need to be changed but just noticed the mismatch.

Comment (jenna-tomkinson) [reply 5_analyze_data/notebooks/KS-test_analysis/visualize_ks_test.r]: I noticed you do change the variable names below this.

**Review by jenna-tomkinson — APPROVED** [review]
LGTM! Very cool looking figures that I would love to discuss more with you later!

Only one small comment from me!

File: 5_analyze_data/notebooks/KS-test_analysis/visualize_ks_test.r (hunk 1 above)
Comment (gwaybio) [reply 5_analyze_data/notebooks/KS-test_analysis/visualize_ks_test.r]: good point! I will change

Comment (gwaybio) [reply 5_analyze_data/notebooks/KS-test_analysis/visualize_ks_test.r]: fixed in a20d3ba918db648a547f92a55520092190e798ab

Going to merge! :rocket:

**Pull request merged**